data/common.yaml

```diff
@@ -1,2 +1,12 @@
 ---
+lookup_options:
+  pam::access::users:
```

Comment:
Should there be a unit test demonstrating how to use the merge in a realistic and reasonable way?